dartagnan/src/main/java/com/dat3m/dartagnan/program/Thread.java

```diff
@@ -22,23 +24,32 @@ public Thread(String name, FunctionType funcType, List<String> parameterNames, i
         Preconditions.checkNotNull(entry, "Thread entry event must be not null");
         this.optScopeHierarchy = Optional.empty();
         this.optSyncSet = Optional.empty();
```

Comment:
also remove the `opt` from the field names

Reply:
Removed